Snowflake.Data.Tests/IntegrationTests/ConnectionChangedSessionIT.cs

```diff
@@ -0,0 +1,160 @@
+using NUnit.Framework;
+using Snowflake.Data.Client;
+using Snowflake.Data.Core;
+using Snowflake.Data.Core.Session;
+using Snowflake.Data.Tests.Util;
+
+namespace Snowflake.Data.Tests.IntegrationTests
+{
+    [TestFixture]
+    [NonParallelizable]
+    public class ConnectionChangedSessionIT : SFBaseTest
```

Comment:
maybe ConnectionMultiplePoolsChangedSessionIT? it would be more visible that it is a test for the new pool